cmake/AddEventLibrary.cmake

```diff
@@ -13,8 +13,8 @@ endmacro()
 macro(generate_pkgconfig LIB_NAME)
     set(prefix      ${CMAKE_INSTALL_PREFIX})
     set(exec_prefix ${CMAKE_INSTALL_PREFIX})
-    set(libdir      ${CMAKE_INSTALL_PREFIX}/lib)
-    set(includedir  ${CMAKE_INSTALL_PREFIX}/include)
+    set(libdir      ${CMAKE_INSTALL_LIBDIR})
```

Comment:
Likely here `${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}` should be used. I will fix this.

Reply:
I might want to use `${CMAKE_INSTALL_FULL_LIBDIR}` instead.

Reply:
Didn't know about this, thanks, will try